gradio/blocks.py

```diff
@@ -88,6 +89,7 @@ def set_event_trigger(
                 "trigger": event_name,
                 "inputs": [block._id for block in inputs],
                 "outputs": [block._id for block in outputs],
+                "js": fn if js else False,
```

Comment:
Interesting, so this is either the serialized javascript function or False? 

Does line 85 need to be changed in the case of a javascript function?

Reply:
Now it's just a string function.